src/test/run-make/raw-dylib-stdcall-ordinal/lib.rs

```diff
@@ -0,0 +1,20 @@
+#![feature(raw_dylib)]
+
+#[link(name = "exporter", kind = "raw-dylib")]
+extern "stdcall" {
+    #[link_ordinal(15)]
+    fn imported_function_stdcall(i: i32);
```

Comment:
Would it make sense to add more parameters (and with different type sizes) to these test functions in order to have more diverse test cases?

Reply:
I could certainly do that, but I think `src/test/run-make/raw-dylib-alt-calling-convention` already covers that pretty well.  The underlying assumption here is that calling conventions and import-by-ordinal are largely orthogonal features, except for making sure that we generate the correct syntax for the .DEF file.  However, if you don't feel that's sufficient, I'm happy to add more test cases here.

(As a side note, I'm aware that raw-dylib-alt-calling-convention is currently ignored on all platforms, because I screwed up the directives at the beginning of the file.  I know how to fix that, and I'll create another PR to do that as soon as this PR is completed.  Meanwhile, I've been running a fixed version of it locally on i686-pc-windows-{msvc,gnu} as I've been working on this PR, and it's worked fine.  Alternatively, if you'd prefer me to fix that test as part of this PR, I'm happy to do that as well.)

Reply:
If this is covered by another test that's OK then. Thanks for clarifying!